### ISSUE TITLE
exception at TEdit.ViewModel.WorldViewModel.PasteChestItem

### ISSUE THREAD

**Issue by TeeTeeHaa** [TEdit/Terraria-Map-Editor#1596]
**Describe the bug**
Trying to add an item to a chest caused an exception:
```
01-24-2022 17:35:06: Error - Object reference not set to an instance of an object.
   at TEdit.ViewModel.WorldViewModel.PasteChestItem(Item item) 
```
TEdit froze shortly after OK-ing the error message window.

**Steps to reproduce the behavior:**
Sorry, seems not to happen again immediately when repeating the progress.

**Expected behavior**
No exception and no freeze.

**Desktop:**
 - OS: Windows 7 64 Bit
 - TEdit Version 4.7.3
 - Terraria Version 1.4.3.2

**Comment by TeeTeeHaa** [comment]
world file:
[issue-1596.wld.zip](https://github.com/TEdit/Terraria-Map-Editor/files/7927991/issue-1596.wld.zip)

log file:
[TEditLog_20220124170826.txt](https://github.com/TEdit/Terraria-Map-Editor/files/7927995/TEditLog_20220124170826.txt)

**Comment by TeeTeeHaa** [comment]
I do not know whether this issue is helpful for you or not. If not then please just close the issue.

P.S.: TEdit is great and really helped me after our _"yeah, now we are in hard mode, but, whoa, why is Crimson and Hallow spreading so fast everywhere, argh, help, panic, no I won't play like that"_ moment.

**Comment by BinaryConstruct** [comment]
Are you using any mods? Thanks for the world file, that will help.

**Comment by TeeTeeHaa** [comment]
No mods in Terraria. It is the "plain" version on Steam.

I found a way to reproduce this:
* Have something in the Windows clipboard. I my case it is the URL of some website. I am not sure whether this step is actually necessary.
* Place a regular chest ("sprite") in the world.
* Select the arrow tool and right-click the chest to edit its contents.
* Do *not* select a slot in the chest.
* Under "Edit Selected Item" enter some characters in the text box for the name and select an item from the appearing list. I used a "Celestial Starboard".
* Click the "Paste" button to the right of the text box.
* Close error message dialog.
* Watch Windows dialog "TEdit has stopped working"...
